### PR TITLE
Kdhaduk tap 1.0.2

### PR DESCRIPTION
**Pull request by dhadukk**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: install-tanzu-cli.md [hunk 1]
```diff
@@ -407,7 +520,7 @@ Follow these instructions to update the Tanzu CLI that was installed for a previ
 3. Navigate to [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform/)
 on Tanzu Network.
 
-4. Click the **tanzu-cli-v0.11.1** directory.
+4. Click the **tanzu-cli-tap-1.0.2** directory.
```
Comment (mdc81) [inline install-tanzu-cli.md:523]: Has the name of cli bundle changed from v<version> to tap-<version>? Also, in all the versions mentioned, we seem to prefix "v", but here it is removed. Is it intentional?

**Comment by RichardJJG** [comment]
@piachakra, we're seeing disagreement here. Please let us know when we can merge.

File: install-tanzu-cli.md (hunk 1 above)
Comment (dhadukk) [reply install-tanzu-cli.md]: changed directory back to tanzu-cli-v0.11.1.

**Comment by dhadukk** [comment]
@RichardJJG @piachakra This PR is ready to be merged.

**Pull request merged**